translation/swift/google/api/annotations.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
Can you also remove all these `.proto` files? 
As they are likely to be outdated over time

Reply:
Removed and updated the steps in README